### ISSUE TITLE
a11y inspect: contrast report should either show AA or AAA, not both, aka contextually shown

### ISSUE THREAD

**Issue by argyleink** [GoogleChromeLabs/ProjectVisBug#172]
currently does a blanket report of contrast, when it should be contextual to font size

**Comment by peculiar-af** [comment]
I submitted #204 to address contextually displaying contrast testing results based on WCAG's guidelines for normal and large text. 

This guideline is chosen based on font size and weight criteria, and is then used to compute a AA and AAA rating for the rules that pertain to that size classification of the selected text, so that the metatip can display the relevant pair of results.
